Symfony/CS/Tests/UtilsTest.php

```diff
@@ -17,21 +17,24 @@
 /**
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  * @author Graham Campbell <graham@mineuk.com>
+ * @author Odín del Río <odin.drp@gmail.com>
  */
 class UtilsTest extends \PHPUnit_Framework_TestCase
 {
     /**
      * @dataProvider provideCamelCaseToUnderscoreCases
+     *
+     * @param string $expected Camel case string.
+     * @param string $input    Input string.
      */
-    public function testCamelCaseToUnderscore($expected, $input = null)
+    public function testCamelCaseToUnderscore($expected, $input)
```

Comment:
Ok, I'm going to make the change, but, it would be fine to have the standard in some readme file ;)
Also, I'm going to replace my test case with same input and expected output in order to satisfy the condition at least in one case.